moonbeam-types-bundle/index.ts

```diff
@@ -390,16 +430,36 @@ export const moonbeamDefinitions = {
     },
     {
       minmax: [800, undefined],
```

Comment:
@jacogr , I'm having difficulties with this.
The scale-info was introduced in runtime 900, so I thought that putting `minmax: [800, 899]` would allow polkadotjs to not use those types for runtime 900 and more, however polkadotjs (6.7.1) fails with:
```
 Error: createType(AccountId):: Invalid AccountId provided, expected 32 bytes, found 20
 ```
 which means that it doesn't take any types from what the node is returning. 
 Am I missing something ?

Reply:
My understanding is the same as yours. I can only think of checking for setup mistakes. Are you sure it's really running 900? And returning metadata v14? No types pasted manually into Apps? etc...

Reply:
Check 6.7.2 - there was an issue, that version fixes it. (Tested against the Moonbeam test network with all types removed and it does register/use the correct versions)

Reply:
Thank you it fixed it